arbnode/espresso/chain/exponential_delay.go

```diff
@@ -0,0 +1,63 @@
+package chain
```

Comment:
same for this file, it should be put under system tests because this will only be used for testing

Reply:
We can move these utilities under the `system_tests` folder.  I would likely still keep it separated from the rest of the files under its own dedicated `espresso` folder to make it clear that is a purely additive change made by us to the original code base, and avoids namespace collisions within the package.